### ISSUE TITLE
 Uncaught Error: Cannot find module "angular-bluebird-promises"

### ISSUE THREAD

**Issue by Zergus** [mattlewis92/angular-bluebird-promises#27]
Hello,

I'm trying to write unit tests (karma) for my angular application. I'm trying to create test module with 'mwl.bluebird' dependency. I'm using webpack to build test files. When I'm trying to import your module:

`import 'angular-bluebird-promises';`

I'm receiving  Uncaught Error: Cannot find module "angular-bluebird-promises".
Reason is:
`ERROR in ./~/angular-bluebird-promises/dist/angular-bluebird-promises.js
Module build failed: ReferenceError: Unknown plugin "add-module-exports"`.

Please add it to package.json. Thanks.

**Comment by mattlewis92** [comment]
I think this is because your webpack build is trying to transpile this module, I'd recommend excluding node_modules in your webpack config:
```
loaders: [{
      test: /.*\.js$/,
      loader: 'babel',
      exclude: /node_modules/
    }]
```

But I'll push a new release shortly which excludes the `.babelrc` from being published which will fix your problem.

**Comment by mattlewis92** [comment]
Should be fixed now in 1.0.2, thanks for reporting!
